picard/metadata.py

```diff
@@ -59,11 +59,19 @@ def images_to_be_saved_to_tags(self):
         if config.setting["save_only_front_images_to_tags"]:
             # FIXME : rename option at some point
             # Embed only ONE front image
-            for img in images:
-                if img.is_front_image():
-                    return [img]
+            front_image = self.get_single_front_image(images)
+            if front_image != []:
+                return front_image
         return images
 
+    def get_single_front_image(self, images=[]):
```

Comment:
Using `images=[]` here may not have the expected result: https://pythonconquerstheuniverse.wordpress.com/2012/02/15/mutable-default-arguments/